composer.json

```
@@ -1,13 +1,13 @@
 {
-    "name": "inpsyde/fau-degree-program-common",
+    "name": "rrze-webteam/fau-studium-common",
```

Let's use `rrze` as the vendor name.
```suggestion
    "name": "rrze/fau-studium-common",
```